spec/classes/config_spec.rb

```diff
@@ -95,7 +99,7 @@
           is_expected.to contain_file('/var/log/marathon')
             .with_ensure('directory')
           is_expected.to contain_file('/etc/marathon/logback.xml')
-            .with_content(/<file>\/var\/log\/marathon\/marathon.log<\/file>/)
+            .with_content(%r{<file>/var/log/marathon/marathon.log</file>})
```

Comment:
Why is this a regex if we're matching a static string? Is that the only way to match a substring or something?